windows/src/desktop/kmshell/xml/keyman_update.xsl

```diff
@@ -3,12 +3,6 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
   <xsl:template name="content_update">
 
```

Comment:
Keep in touch was removed in earlier PRs this code need to go then. But this still being here was causing the keyboard list dropdowns to stop working after checkout for updates.